server/queries.go

```diff
@@ -21,8 +22,8 @@ type QueryRequest struct {
 // QueriesRequest converts all queries to queryConfigs with the help
 // of the template variables
 type QueriesRequest struct {
-	Queries      []QueryRequest          `json:"queries"`
-	TemplateVars chronograf.TemplateVars `json:"tempVars,omitempty"`
+	Queries      []QueryRequest           `json:"queries"`
```

Comment:
Hmm this kind goes against what I was saying with the `Shifts`->`TimeShifts` thing.